user_guide/common_features.rst

```diff
@@ -49,10 +49,32 @@ prior agreements.
 
 In OroCRM, all of this data is tracked using three types of records: channels, customers, and accounts.
 
+
+Understanding Accounts, Channels and Customers
+----------------------------------------------
+
+For each :ref:`channel created <user-guide-channel-guide-create>`, there is a set of 
+:term:`entities <Entity>` defined. These entities correspond to types of information collected from the corresponding 
+data source.
```

Comment:
Edit: For each :ref:`channel that is created <user-guide-channel-guide-create>`, there is a set of :term:`entities <Entity>` that are defined. These entities correspond to the various types of information collected from the corresponding data source.